src/resources/i18n/emailpro/Messages_fr_FR.xml

```diff
@@ -115,5 +115,7 @@
    <translation id="emailpro_tab_domain_add_failure" qtlid="375972">Une erreur est survenue lors de l'ajout du domaine à votre serveur E-mail Pro.</translation>
    <translation id="emailpro_tab_domain_diagnostic_mx_no_antispam_config" qtlid="375985">pro1.mail.ovh.net</translation>
    <translation id="emailpro_ACTION_add_domain_invalid_domain" qtlid="375998">Le domaine saisi n'est pas valide</translation>
+   <translation id="emailpro_ACTION_order_accounts_step1_price">{0} HT ({1} TTC)</translation>
+   <translation id="emailpro_ACTION_order_accounts_step1_price_germany">{0} ({0} netto)</translation>
```

Comment:
Should be somthing like : 
`<translation id="emailpro_ACTION_order_accounts_step1_price_with_tax">{0} ({0} TTC)</translation>`

Reply:
Or rather something like `{0} ({1} TTC)`?

Reply:
Updated the translation to <translation id="emailpro_ACTION_order_accounts_step1_price_with_tax">{0} ({0} TTC)</translation>

Thanks for pointing it out :)